python/tvm/script/parse/tir/tir.py

```diff
@@ -80,6 +81,48 @@ def visit_assign(self: Parser, node: doc.Assign) -> None:
         self.eval_assign(target=lhs, source=rhs, bind_value=bind_value)
 
 
+@dispatch.register(token="tir", type_name="AugAssign")
+def visit_aug_ssign(self: Parser, node: doc.AugAssign) -> None:
+    lhs_pos = (
+        node.target.lineno,
+        node.target.col_offset,
+        node.target.end_lineno,
+        node.target.end_col_offset,
+    )
+    rhs_pos = (
+        node.value.lineno,
+        node.value.col_offset,
+        node.value.end_lineno,
+        node.value.end_col_offset,
+    )
+    node.target.ctx = doc.Load(*lhs_pos)
+    lhs_name = f"__tvm_tmp_value_aug_assign_lhs_{lhs_pos[0]}"
+    rhs_name = f"__tvm_tmp_value_aug_assign_rhs_{rhs_pos[0]}"
+    lhs = self.eval_expr(node.target)
+    rhs = self.eval_expr(node.value)
+    self.var_table.add(lhs_name, lhs)
+    self.var_table.add(rhs_name, rhs)
```

Comment:
remember to remove those two variables after evaluation...alternatively, shall we have a new var frame?

Reply:
yes, I have added a temp `with` frame for `var_table`